pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/export/tabular/TabularParticipantExporter.java

```diff
@@ -42,29 +50,74 @@ public static TabularParticipantExporter getExporter(List<ModuleExportConfig> mo
         return new TsvParticipantExporter(moduleConfigs, participantValueMaps, fileFormat);
     }
 
+    /** returns a filename such that alphabetical sorting will also put them in chronological order */
     protected static String getExportFilename(String suffix) {
         LocalDate date = LocalDate.now();
         DateTimeFormatter formatter = DateTimeFormatter.ofPattern(FILE_DATE_FORMAT);
         String exportFileName = String.format("Participant-%s.%s", date.format(formatter), suffix);
         return exportFileName;
     }
 
+    /**
+     * Gets all the column names that need to be rendered for a given config, including repeats, details, and/or
+     *  splitting options across multiple column.  The column name is based off the stable name of the associated item
+     *  and needs to be globally unique
+     */
+
     public static List<String> getAllColumnNames(FilterExportConfig filterConfig, int formRepeatNum, int questionRepeatNum) {
+        List<String> allColNames = new ArrayList<>();
+        List<String> optionStableIds = Collections.singletonList(null);
```

Comment:
`Collections.emptyList()` is better alternative maybe?

Reply:
This is a bit unintuitive, but I actually want one iteration even if there are no options.  I'll see if I can refactor the code around it -- you're right that it's not clear at all